packages/preview-middleware-client/src/adp/ui5-version-utils.ts

```diff
@@ -0,0 +1,53 @@
+
+/**
+ * Checks if the given version is lower than the required minimal version.
+ * @param version to check
+ * @param minVersion to check
+ *
+ * @returns boolean
+ */
+export function isLowerThanMinimalUi5Version(version: string, minVersion: string): boolean {
+    if (version !== undefined && version !== '' && minVersion !== undefined && minVersion !== '') {
+        const minVersionParsed = parseUI5Version(minVersion);
+        const ui5VersionParsed = parseUI5Version(version);
+        if (
+            !isNaN(ui5VersionParsed.major) &&
```

Comment:
For this big if statement could we split it more for readability? First, check if the version and minVersion are not empty. Second, check ui5VersionParsed.minor and ui5VersionParsed.major that they are valid numbers. Then check in the second if statement, which will be inside the first one, all the version stuff.

Something like this:

```typescript
if (version && minVersion) {
    const minVersionParsed = parseUI5Version(minVersion);
    const ui5VersionParsed = parseUI5Version(version);

    if (!isNaN(ui5VersionParsed.major) && !isNaN(ui5VersionParsed.minor)) {
        if (ui5VersionParsed.major < minVersionParsed.major) {
            return true;
        }
        if (ui5VersionParsed.major === minVersionParsed.major && ui5VersionParsed.minor < minVersionParsed.minor) {
            return true;
        }
    }
}
```

This also does not take into account the patch version (not sure if this is needed).

Reply:
Done